### PR TITLE
Avatar

### PR DESCRIPTION
**Pull request by raghav6202**
Please review the P.R of Avatar component
image

### CODE REVIEW

File: Components/Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,136 @@
+@import url("landingPage.css");
+
+
+.avatar-component
+{
+    display: flex;
+    flex-flow: column;
+    padding: 0 1rem;
+
```
Comment (zabihshaik) [inline Components/Avatar/avatar.css:9]: Can avoid the blank spaces in between properties and }

File: Components/Avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+{
+    font-size: 1.8rem;
+    color: var(--secondary-color);
+}
+.info-text
+{
+    font-size: 1.4rem;
+}
+.component-display-container
+{
```
Comment (zabihshaik) [inline Components/Avatar/avatar.css:40]: Can Use Prettier for Formatting / Beautifying Code

File: Components/Avatar/avatar.html [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+<p class="info-text">Avatar is available in 4 different sizes. You can use image in Avatar. You need to include
+     these classes in the image tag.Choose the appropriate classes according to the required size <b> img-avatar xl, img-avatar l, img-avatar m, img-avatar s (e.g. class="img-avatar m")</b>
+    Don't forget to add Responsive Image class names for img element.</p>
+
+
+</div>
+
+<div class="component-display-container" id="image-avatar-component">
+
+<img class="img-avatar xl" src="https://media.istockphoto.com/photos/lion-predator-dangerous-mane-picture-id1287214055?s=612x612" alt="avatar image">
```
Comment (zabihshaik) [inline Components/Avatar/avatar.html:35]: Can Use Prettier for Formatting / Beautifying Code (Space from the left side for a child element)

File: Components/Avatar/landingPage.css [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+{
+    width:50%;
+}
+
+}
+
+  
+
+@media screen and (max-width : 600px){
+
```
Comment (zabihshaik) [inline Components/Avatar/landingPage.css:212]: Remove the unnecessary blank spaces in between

**Comment by zabihshaik** [comment]
1. Can Use Prettier for Formatting / Beautifying Code (To know the proper differentiation between parent and it's child element) 
2. Remove unnecessary blank spaces in between properties and }
Other than that everything is good.

File: Components/Avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+}
+
+.component-heading
+{
+    font-size: 3rem;
+    color: var(--secondary-color);
+    font-weight: 400;
+}
+
+.component-subHeading
```
Comment (TejasShekar) [inline Components/Avatar/avatar.css:29]: Try not to name classes using camelCase. Always maintain consistency in naming classes. Follow one naming convention for consistency.

File: Components/Avatar/avatar.html [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+<div class="component-subHeading">Image Avatar</div>
+
+<p class="info-text">Avatar is available in 4 different sizes. You can use image in Avatar. You need to include
+     these classes in the image tag.Choose the appropriate classes according to the required size <b> img-avatar xl, img-avatar l, img-avatar m, img-avatar s (e.g. class="img-avatar m")</b>
+    Don't forget to add Responsive Image class names for img element.</p>
+
+
+</div>
+
+<div class="component-display-container" id="image-avatar-component">
```
Comment (TejasShekar) [inline Components/Avatar/avatar.html:33]: I couldn't find any id selector in your CSS file. Please check.

**Review by TejasShekar — COMMENTED** [review]
Looks great. 
My suggestions :
- try to give better class names that are self descrptive.
- use prettier to format your code better. 
- set font-size: 62.5% in root/html selector so that your rem calculations will be simple.

File: Components/Avatar/landingPage.css (hunk 4 above)
Comment (raghav6202) [reply Components/Avatar/landingPage.css:212]: Thank you will look into it

File: Components/Avatar/avatar.css (hunk 5 above)
Comment (raghav6202) [reply Components/Avatar/avatar.css:29]: Will keep that in mind

**Pull request merged**